### ISSUE TITLE
[UI/UX]: The hero section's heading, body texts and icons have colour contrast, padding and responsiveness issues

### ISSUE THREAD

**Issue by reach2saksham** [AOSSIE-Org/Website#713]
Problems: 
- From md (768px) to lg (1024px) viewport, the landing page structure is in such a way that if a user with tablet devices or devices within the given viewports, would not be able to even read the heading "We Educate We Innovate", especially in light mode.

<img width="760" height="759" alt="Image" src="https://github.com/user-attachments/assets/95e4886b-af5e-4aeb-a5f1-974e1bc489a7" />

- The heading "We Educate We Innovate" currently lies at the edge of its background. It would be very suitable to to let it have some bottom padding.
- The current applied colour for body text and icons is text-zinc-500, which lacks sufficient contrast with its background, causing accessibility issues. According to WCAG Compliance, the ratio should be at least 4.5:1

<img width="1217" height="792" alt="Image" src="https://github.com/user-attachments/assets/a31cfed1-d381-4adf-868d-2a8e88d260d7" />

Solutions: 

- Either bring enough contrast in the heading or let it have consistenet background within the viewports
- Provide padding below "We Educate We Innovate"
- Replace text-zinc-500 with text-zinc-800 in only over lg (because its background is yellow). text-zinc-500 have enough contrast with white
